tests/general/printf-no-long-double.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+volatile double x = 42.0;
```

Comment:
Is volatile needed here?

Reply:
Just an abundance of caution; the point is to test the `libc` printf; if a compiler ever somehow constant-folds this into the printf string, that would defeat the point.